nipype/interfaces/dtitk/utils.py

```diff
@@ -9,14 +20,14 @@
 class TVAdjustOriginInputSpec(CommandLineInputSpec):
```

Comment:
Getting rid of this because it is redundant (this calls the same command as TVAdjustVoxelspace)

Reply:
As long as everything that TVAdjustOrigin does can be done with TVAdjustVoxelspace, that sounds fine to me.

Reply:
Yes. It calls the command TVAdjustOrigin with a specific set of flags, so it is redundant as an interface.